web/src/app/browser/src/contextManager.ts

```diff
@@ -58,4 +73,61 @@ export default class ContextManager extends ContextManagerBase {
     }
     return false;
   }
+
+  /**
+   * Reflects the active 'target' upon which any `set activeKeyboard` operation will take place.
+   * When `null`, such operations will affect the global default; otherwise, such operations
+   * affect only the specified `target`.
+   */
+  protected get keyboardTarget(): OutputTarget {
+    // Remove `&& false` once the inlined section below is implemented.
```

Comment:
```suggestion
    // TODO: Remove `&& false` once the inlined section below is implemented.
```

Just makes it clearer that this is not yet finished